cloudbuild.yaml

```
@@ -91,6 +95,8 @@ steps:
     #!/bin/sh
     set -o errexit
     set -o xtrace
+    docker tag bazel/java:java_base_debian9                 gcr.io/$PROJECT_ID/java-debian9:base
+    docker tag bazel/java:java_base_debug_debian9           gcr.io/$PROJECT_ID/java-debian9:base-debug
```

For consistency,
```suggestion
    docker tag bazel/java:java_base_debian9                 gcr.io/$PROJECT_ID/java:base
    docker tag bazel/java:java_base_debian9                 gcr.io/$PROJECT_ID/java-debian9:base
    docker tag bazel/java:java_base_debug_debian9           gcr.io/$PROJECT_ID/java:base-debug
    docker tag bazel/java:java_base_debug_debian9           gcr.io/$PROJECT_ID/java-debian9:base-debug
```
Reorder these if you want.